lib/install/development.rb

```diff
@@ -0,0 +1,152 @@
+def write_redis_recommendation(development, lines, index)
+  gemfile = Rails.root.join("Gemfile")
+  redis_pattern = /gem ['"]redis-session-store['"]/
+
+  # add redis-session-store to Gemfile, but comment it out
+  if !File.read(gemfile).match?(redis_pattern)
+    append_file gemfile, "\n# Use Redis for session storage", verbose: false
+    gem "redis-session-store", "0.11.4"
```

Comment:
```suggestion
    append_file gemfile, <<RUBY
# Use Redis for session storage",
gem "redis-session-store", "0.11.4"
RUBY
```

Reply:
I'm on the fence about converting this to a heredoc, though you're probably right. It all comes down to the subtle difference in the output on the console - I like the output that running the `gem` action has.

I know, it's a small thing. 😁

Reply:
Yeah, I feel you 🙂